plumber_rest_api/api/handlers/insert.R

```diff
@@ -26,7 +26,7 @@ handler <- function(data, token, id, bucket){
   values <- tryCatch ({
     unserializeJSON(data)
   }, error = function(err) {
-    log_error("Couldn't serialize json")
+    log_error("Couldn't unserialize json")
     "Couldn't serialize json"
```

Comment:
I think this is redundant in a way that we will show a log error then return a string.

Reply:
You're right! moved that to reuse the message below